server/api/orders.js

```diff
@@ -1,39 +1,55 @@
 const router = require('express').Router();
-const { Order, LineItem } = require('../db/models');
-const { requireAdmin, requireLogin, requireUserOrAdmin } = require('./util');
+const {Order, LineItem, Product, Image} = require('../db/models');
+const {requireAdmin, requireLogin, requireUserOrAdmin} = require('./util');
 
 //Get single order
 
-router.get(
-  '/:id',
-  requireLogin,
-  requireUserOrAdmin,
-  (req, res, next) => {
-    return Order.findById(req.params.id)
-      .then(order => res.json(order))
-      .catch(next);
-  }
-);
+router.get('/', requireLogin, requireUserOrAdmin, async (req, res, next) => {
```

Comment:
we think that this should be open for admins only. Also when we use requireUserOrAdmin, it uses req.params.id which doesn't exist right now.